airbyte-integrations/connectors/source-trustpilot/README.md

```diff
@@ -1,36 +1,12 @@
 # Trustpilot Source
 
-This is the repository for the Trustpilot source connector, written in Python.
+This is the repository for the Trustpilot source connector.
 For information about how to use this connector within Airbyte, see [the documentation](https://docs.airbyte.com/integrations/sources/trustpilot).
 
 ## Local development
 
-### Prerequisites
```

Comment:
Can you replace this with the blob on installing with Poetry? Any currently certified Python connector should have it.